### PR TITLE
Complete rewrite for Rust 1.3.0 and Cargo

### PR DESCRIPTION
**Pull request by crossroads1112**

### CODE REVIEW

**Comment by crossroads1112** [comment]
Wow, I didn't realize how amazing rustdoc is. Clone my repo and run `cargo doc --open`

It's pretty awesome.

File: Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[package]
+name = "rustcheck"
+version = "0.1.0"
+authors = ["Chad Sharp <crossroads1112@gmail.com>"]
```
Comment (mcandre) [inline Cargo.toml]: Would you mind adding me to the list?

Andrew Pennebaker
andrew.pennebaker@gmail.com

**Comment by crossroads1112** [comment]
I apologize, that is automatically generated by Cargo from my git config. I'm on mobile right now, but when I get back to my laptop I will

**Comment by crossroads1112** [comment]
Fixed

**Comment by mcandre** [comment]
Thanks for putting in so much work!

**Pull request merged**